cluster-autoscaler/core/scaledown/actuation/update_latency_tracker.go

```diff
@@ -0,0 +1,100 @@
+package actuation
+
+import (
+	"time"
+
+	"k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/taints"
+	"k8s.io/klog/v2"
+)
+
+const sleepDurationWhenPolling = 50 * time.Millisecond
+const watiForTaintingTimeoutDuration = 5 * time.Minute
+
+type NodeTaintStartTime struct {
+	nodeName  string
+	startTime time.Time
+}
+
+type UpdateLatencyTracker struct {
```

Comment:
Please capitalize all members that are part of the public interface of `UpdateLatencyTracker`. Right now it is hard to tell from this code which fields are meant to be directly modified and which ones are internal state of the object.

Reply:
Done